mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxManagerTest.java

```diff
@@ -63,165 +66,136 @@
 
 import reactor.core.publisher.Mono;
 
-class StoreMailboxManagerTest {
+interface MailboxManagerContract {
```

Comment:
The switch to an interface could have been split into a separate commit to make review easier.